docs/manage/fleet/provision/end-user-setup.md

```diff
@@ -135,6 +138,59 @@ Note that any features that require internet access will not function if the con
 {{% /tablestep %}}
 {{< /table >}}
 
+## Set up your machine using a custom Flutter app
+
+If you are building your own app to provide provisioning functionality you have three options for provisioning:
+
+| Provisioning method | Description | Notes | Package |
+| ------------------- | ----------- | ----- | ------- |
+| **Bluetooth with WiFi** | Ask the user to connect to the machine over Bluetooth. The user then provides network credentials for an internet-connected WiFi network, through which machine setup can then occur. | Recommended, if available. | [Example](https://github.com/viamrobotics/viam_flutter_provisioning/) |
+| **WiFi** | Ask the user to connect to the machine's temporary WiFi network. The user then provides network credentials for an internet-connected WiFi network, through which machine setup can then occur. | Slower than Bluetooth with WiFi but faster than Bluetooth tethering. | TODO |
```

Comment:
@kevin49999 do you mind taking another look here:
1. Is this table covering this correctly?
2. Do we have an example for WiFi?

Table is easier to look at here: https://deploy-preview-4372--viam-docs.netlify.app/manage/fleet/provision/end-user-setup/#set-up-your-machine-using-a-custom-flutter-app